src/StatusQ/Components/StatusChatListCategoryItem.qml

```diff
@@ -23,15 +23,12 @@ StatusListItem {
     property alias menuButton: menuButton
     property alias toggleButton: toggleButton
 
-    signal clicked(var mouse)
     signal addButtonClicked(var mouse)
     signal menuButtonClicked(var mouse)
     signal toggleButtonClicked(var mouse)
 
     color: sensor.containsMouse || highlighted ? Theme.palette.baseColor2 : "transparent"
 
-    sensor.onClicked: statusChatListCategoryItem.clicked(mouse)
-
```

Comment:
Why are these removed? This is not related to the scope of this PR, plusm we need those.

Reply:
> Why are these removed? This is not related to the scope of this PR, plusm we need those.

Since we have now "clicked" singla in `StatusListItem` and `StatusChatListCategoryItem` is inherited from `StatusListItem` we don't need it in `StatusChatListCategoryItem`, that works the same way now as it worked before.
that's because we have them now in StatusListItem